damus/Views/PostView.swift

```diff
@@ -84,8 +84,10 @@ struct PostView: View {
         let imagesString = uploadedMedias.map { $0.uploadedURL.absoluteString }.joined(separator: " ")
         
         let img_meta_tags = uploadedMedias.compactMap { $0.metadata?.to_tag() }
-
-        content.append(" " + imagesString + " ")
+        
+        if !imagesString.isEmpty {
```

Comment:
Another solution would be to keep this line content.append(" " + imagesString + " ") before line#82 which is var content = self.post.string.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)